src/CMakeLists.txt

```diff
@@ -426,13 +426,13 @@ add_dependencies(common-objs legacy-option-headers)
 
 if(WITH_JAEGER)
   find_package(yaml-cpp 0.6.0)
+  execute_process(COMMAND bash -c "sed -i 's/#//' debian/libjaeger.install"
```

Comment:
please use build profile to manage this optional package.

1. add `Build-Profiles: <pkg.ceph.jaeger>` at the end of `Package: libjaeger` section in `debian/control`.
2. add `<pkg.ceph.jaeger>` at the end of all jaeger dependencies  in `debian/control`.
3. check `pkg.ceph.jaeger` in `debian/rules`, like
  ```sh
  ifeq ($(filter pkg.ceph.jaeger,$(DEB_BUILD_PROFILES)),)
    extraopts += -DWITH_JAEGER=OFF
  else
    extraopts += -DWITH_JAEGER=ON
  endif
  ```
4. append `,pkg.ceph.jaeger` to `build_profiles` in `install-deps.sh` in `install-deps.sh` if `WITH_JAEGER` environment variable is set.

see https://wiki.debian.org/BuildProfileSpec

Reply:
thanks, that's a good idea! updated :)